### PR TITLE
Fix spelling for some insults

### PR DESCRIPTION
**Pull request by bugybunny**
(╯°□°）╯︵ ┻━┻)

### CODE REVIEW

**Comment by dbrgn** [comment]
hm, why did you need to remove the space?

**Comment by bugybunny** [comment]
Oh haha, I forgot the rules :D I thought there is no space, the military is damaging my brain already.

File: one_liners.js [hunk 1]
```diff
@@ -30,13 +30,13 @@ const insults = [
...
   'Zu Schade, dass DAS überhaupt niemanden interessiert, %u%.',
-  'Sollt’ ich in deiner Nähe sterben,%u%, möcht’ ich, daß man mich desinfiziert!',
+  'Sollt’ ich in deiner Nähe sterben, %u%, möcht’ ich, dass man mich desinfiziert!',
   'Bist du das, %u%? Es riecht hier so nach Jauche und Dung!',
   'Ist der Blick in den Spiegel für Dich jeden Tag nicht eine Erniedrigung, %u%?',
   'Du hast soviel Sexappeal wie ein Croupier, %u%.',
   'Dein Geplänkel kommt nicht richtig in Schwung, %u%!',
   'Wurdest du damals von einem Schwein adoptiert, %u%?',
-  'Das ich nicht lache, %u%',
+  'Dass ich nicht lache, %u%',
```
Comment (rnestler) [inline one_liners.js:39]: I think this one was correct before, wasn't it?

Comment (rnestler) [reply one_liners.js:39]: Hmm never mind, according to https://www.korrekturen.de/forum.pl/md/read/id/10074/sbj/das-oder-dass-ich-nicht-lache/ it is "Dass".

**Pull request merged**